library/src/main/java/com/bumptech/glide/load/resource/gif/GifDrawable.java

```diff
@@ -309,7 +309,23 @@ public void setLoopCount(int loopCount) {
         }
 
         if (loopCount == LOOP_INTRINSIC) {
-            maxLoopCount = decoder.getLoopCount();
+            int netscapeIterationCount = decoder.getNetscapeIterationCount();
```

Comment:
@TWiStErRob 
Thanks for your clarification! I understand.
Your suggested naming also helps me. I follow it.

I'll create a new pull request for this fix.